jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantWidget/ExtendedVariantWidget.js

```diff
@@ -31,71 +33,323 @@ export default jbrowse => {
+                            }
+                            else{
+                                gtKey[gtVal] = alt[gtKey[gtVal]-1]
+                                alleleCounts[gtKey[gtVal]] = alleleCounts[gtKey[gtVal]] + 1 // tick up allele count
+                                alleleTotal = alleleTotal + 1
+                            }
+                        }
+                        gtKey = gtKey[0] + "/" + gtKey[1]         // for the purposes of the chart, phased/unphased can be counted as the same
+                       if(gtCounts[gtKey]){                       // if gtCounts entry is not null, or we have a preexisting entry for it
+                            gtCounts[gtKey] = gtCounts[gtKey] + 1 // increment count for that gt
+                            gtTotal = gtTotal + 1                 // increment our total count
+                        }
+                        else {                                    // else if gtCounts entry is null, or we don't have an entry, set to 1
+                            gtCounts[gtKey] = 1
+                            gtTotal = gtTotal + 1                 // increment our total count
+                        }
+                    }
+                }
+            }
+            var gtBarData = [
+            [
+                'Genotype',
+                'Total Count',
+                { role: 'style' },
+                {
+                  sourceColumn: 0,
+                  role: 'annotation',
+                  type: 'string',
+                  calc: 'stringify',
+                },
+            ]]
+
+            var rounds = 0
+            var decimal = 1
+            while(decimal != 2){
+                for(var entry in gtCounts){
+                    rounds = rounds + round(gtCounts[entry]/gtTotal*100, decimal)
+                }
+                if(rounds == 100){
+                    break
+                }
+                rounds = 0
+                decimal = decimal + 1
+            }
+            for(var entry in gtCounts){
+                var rounded = round(gtCounts[entry]/gtTotal*100, decimal)
+                if(rounds != 100){
+                    rounded = "~" + rounded
+                }
+                gtBarData.push(
+                    [entry, gtCounts[entry], "#0088FF", rounded+"%"]
+                )
+            }
+
+            var alleleTableRows = []
+            for(var allele in alleleCounts){
+                alleleTableRows.push(
+                    <TableRow>
+                            <TableCell>{allele}</TableCell>
+                            <TableCell>{round(alleleCounts[allele]/alleleTotal, 4)}</TableCell>
+                            <TableCell>{alleleCounts[allele]}</TableCell>
+                    </TableRow>
+                )
+            }
+            var gtTitle = "Genotype Frequency ("+gtTotal.toString()+")"
+
+            // TODO - get variables, prepare genotypeTable link
```

Comment:
It's a good idea to move ahead as-is, but let's open an issue specifically on this URL so we dont lose track of it

Reply:
Starter issue created - #63